sdk/src/wallet/operations/reissue.rs

```diff
@@ -16,48 +16,69 @@ use crate::{
     wallet::{types::InclusionState, Error, Wallet},
 };
 
-const DEFAULT_REISSUE_UNTIL_INCLUDED_INTERVAL: u64 = 1;
-const DEFAULT_REISSUE_UNTIL_INCLUDED_MAX_AMOUNT: u64 = 40;
+// Time in milliseconds
+const DEFAULT_AWAIT_TX_ACCEPTANCE_INTERVAL: u64 = 500;
+const DEFAULT_AWAIT_TX_ACCEPTANCE_MAX_AMOUNT: u64 = 80;
 
 impl<S: 'static + SecretManage> Wallet<S>
 where
     Error: From<S::Error>,
     crate::client::Error: From<S::Error>,
 {
-    /// Reissues a transaction sent from the account for a provided transaction id until it's
-    /// included (referenced by a milestone). Returns the included block id.
-    pub async fn reissue_transaction_until_included(
+    /// Checks the transaction state for a provided transaction id until it's accepted. Interval in milliseconds.
+    /// Returns the block id that contains this transaction.
+    pub async fn await_transaction_acceptance(
```

Comment:
bit nitpicky but maybe the `await_` prefix is not so good since it's one of Rust's keywords and `await_transaction_acceptance.await?` reads weird. Therefore I suggest something like:
```rust
async fn block_on_transaction_acceptance(...)
```

Reply:
I don't know about that tbh... that function doesn't really block, that's what await means. Whether it's blocking or not depend on the caller.

Reply:
I don't really care, can also undo, maybe let's wait for an opinion from @DaughterOfMars

Reply:
`await` means that it cannot make immediate process, hence is blocked by some other resource to become available. It could be blocked by some required client request or some lock to become available. Typically if there are other tasks, then the runtime switches to another, which means that the overall application isn't blocked, but I get your point. I called it like that because in most examples in order to continue with it we need to wait for the transaction to get accepted, hence the example is blocked from further continuing until that happened.

Reply:
> I called it like that because in most examples in order to continue with it we need to wait for the transaction to get accepted, hence the example is blocked from further continuing until that happened.

That's exactly what I'm saying, this is a caller behaviour. If `block_on` is in the name, I understand that it doesn't yield control to the scheduler, but it absolutely does here. But maybe it's just me, idk.

Reply:
yup, I agree. That would be confusing. Maybe `wait_on`? :shrug:

Reply:
Please revert :trollface:

Reply:
I do think `WaitFor` is better than `Await` in this case.